python/tank/bootstrap/baked_configuration.py

```diff
@@ -118,13 +118,13 @@ def get_tk_instance(self, sg_user):
         return super(BakedConfiguration, self).get_tk_instance(sg_user)
 
     @classmethod
-    def bake_config_scaffold(cls, path, sg_connection, entry_point, config_descriptor):
+    def bake_config_scaffold(cls, path, sg_connection, plugin_id, config_descriptor):
         """
         Helper method that can be used to generate a baked scaffold in a given path.
 
         :param path: Path to generate scaffold in.
         :param sg_connection: Shotgun API instance
-        :param entry_point: Config entry point
+        :param plugin_id: Config plugin id
```

Comment:
I really like the docs for the `plugin_id` param in previous methods. Maybe use that here as well?